docs/Running-behind-haproxy.md

```diff
@@ -0,0 +1,272 @@
+# Netdata via HAproxy
+
+To proxy requests from [HAProxy](https://github.com/haproxy/haproxy) to Netdata, the following configuration can be used:
+
+## Default Configuration
+
+For all examples, set the mode to `http`
+
+```
+defaults
+    mode    http
+```
+
+## Simple Configuration
+
+A simple example where the base URL is used with no subpath:
```

Comment:
Does this mean they would access Netdata directly at `example.com`?

Reply:
Yes. Does this seem better?

A simple example where the base URL, say http://example.com, is used with no subpath:

Reply:
Great.